Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py

```diff
@@ -174,15 +157,24 @@ def module_test_command(client: Client) -> COMMAND_OUTPUT:
         dict: Operation entry context - Empty.
         dict: Operation raw response - Empty.
     """
-    try:
-        client.get_indicators(limit=4)
-    except RuntimeError:
-        raise DemistoException("Unable to communicate with ThreatConnect API!")
+    url = '/api/v3/groups?resultLimit=4'
```

Comment:
Do not pass query parameters as part of the URL; There is a dedicated argument for it.